File: java/client/src/main/java/glide/api/models/exceptions/ConfigurationError.java
/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
package glide.api.models.exceptions;

/** Errors that are thrown when a request cannot be completed in current configuration settings. */
public class ConfigurationError extends GlideException {
    public ConfigurationError(String message) {
        super(message);
    }
}

File: csharp/sources/Valkey.Glide.InterOp/Errors/ConfigurationError.cs
namespace Valkey.Glide.InterOp.Errors;

/// <summary>
/// An errors that is thrown when a request cannot be completed in current configuration settings.
/// </summary>
public sealed class ConfigurationError : GlideException
{
    // TODO set HelpLink with link to wiki

    public ConfigurationError() { }

    public ConfigurationError(string message) : base(message) { }

    public ConfigurationError(string message, Exception innerException) : base(message, innerException) { }
}